### ISSUE TITLE
Bibliography count is not working properly

### ISSUE THREAD

**Issue by QYQSDTC** [UTRGV-SMSS/Thesis-Template#11]
![image](https://user-images.githubusercontent.com/15608306/78194610-435caf80-7443-11ea-9948-e96999c6d2fd.png)
It shows 0 references in Abstract and I am using natbib for bibliography.

**Comment by ggarza** [comment]
Did you change the document class options from "draft" to "final." 

It could also be that you are using natbib.  The template is not intended to be used with natbib.

As a simple fix, you can edit the file "UTRGVThesis.cls."  Change line 240 that has `\total{bibnum}~references`  to have just the actual number of references instead of `\total{bibnum}`

**Comment by QYQSDTC** [comment]
Yes, I am using "final" and it is the problem with natbib. Are there requirements for the References? if not, I will give it a simple fix.

**Comment by ggarza** [comment]
I believe the best thing to do is to use that simple fix.  

Natbib is using its own counter that is distinct from the `bibnum` counter.  I believe we could also copy the natbib counter to the `bibnum` counter.  Unfortunately, I haven't used natbib so I don't have a good way to test it.
